src/Views/HamburgerButton/HamburgerButton.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import bars from './menu.svg'
+
+export default class HamburgerButton extends React.Component {
+    render(){            
+        return (
+            <button style={{position:"fixed", top: 0, left:0, zIndex: 99999, border: "none", padding: 16, margin:0, boxSizing: "border-box", background: "#fff", width: 50, height: 50, top: 0, left: 0}} onClick={this.props.action} className="btn-toggle-nav">
```

Comment:
For the sake of consistency, its best we keep all css in the css files. It makes code readability easier.

Reply:
Agreed but perhaps we also need to come up with a naming convention... while css classes are imported at component level, the rule would applies if it matches another element in a separate component with the same identifier.